ci.sh

```diff
@@ -20,11 +20,11 @@ function run_tests {
     echo "Testing host architecture"
   fi
 
-  ./miri test --locked
+  ./miri test --locked --features=expensive-debug-assertions
```

Comment:
This will rebuild miri since the original `./miri build` was with a different feature set.

Maybe put these flags into a variable and pass them to all invocations?
In fact, we already have `CARGO_EXTRA_FLAGS="--all-features"`, isn't that enough?

Reply:
Yes I think so. I just didn't notice that in the CI script.